corehq/apps/app_manager/views/apps.py

```diff
@@ -277,12 +277,6 @@ def app_source(request, domain, app_id):
 
 
 @require_can_edit_apps
-def copy_app_check_domain(request, domain, name, app_id_or_source):
```

Comment:
ok. `git log -S copy_app_check_domain` implies it was added in 67d05212fd447e3747ce4b24a1f48588de144e14.

It's super not explicit, but the intention was that `login_and_domain_required` be used to check that the user has login access to the domain `req.GET.get('domain')` as well as the domain passed in the url (/a/:domain:/...). It's not the most straightforward way to do it, but it is an important check, and I think the proposed change would remove that check. Do you agree?